### PR TITLE
Increase article titles length (fixes #37)

### LINKED ISSUE #37: Increase length of title and resolved_title to 1024

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: readinglist/views/article.py [hunk 1]
```diff
@@ -14,14 +14,29 @@ def preparer(self, appstruct):
         return strip_whitespace(appstruct)
 
 
+class URL(SchemaNode):
+    """String representing a URL."""
+    schema_type = String
+    validator = colander.All(colander.url, colander.Length(min=1, max=2048))
```
Comment (Natim) [inline readinglist/views/article.py]: The max should be a configuration.

File: readinglist/views/article.py [hunk 2]
```diff
@@ -14,14 +14,29 @@ def preparer(self, appstruct):
...
+
+    def preparer(self, appstruct):
+        return strip_whitespace(appstruct)
+
+
+class ArticleTitle(SchemaNode):
+    """String representing the title of an article."""
+    schema_type = String
+    validator = colander.Length(min=1, max=1024)
```
Comment (Natim) [inline readinglist/views/article.py]: The max should be a configuration.

File: readinglist/views/article.py (hunk 1 above)
Comment (leplatrem) [inline readinglist/views/article.py]: Changing the value might imply running some kind of migration depending on the backend. Should we allow that from settings?

Using a constant in code base seems enough IHMO

Putting configuration in schemas is painful :)

Comment (Natim) [inline readinglist/views/article.py]: > putting configuration in schemas is painful

Yes I have seen that.

Comment (ametaireau) [inline readinglist/views/article.py]: open for discussion: should we truncate the URL when it's longer than what we want? (rather than rejecting it)

Comment (leplatrem) [inline readinglist/views/article.py]: this question could be handled in a separate issue, since it also concerns article titles (@rnewman proposition)

Comment (Natim) [inline readinglist/views/article.py]: Agreed

**Pull request merged**